erpnext/templates/includes/transaction_header.html

```diff
@@ -1,17 +1,35 @@
 <div class="row margin-top bold">
-    <div class="col-sm-2 ">
-        {{ _("Name") }}
-    </div>
-    <div class="col-sm-3 font-size">
-        {{ _("Transaction Date") }}
-    </div>
-    <div class="col-sm-3">
-        {{ _("Delivery Date") }}
-    </div>
-    <div class="col-sm-2 text-right">
-        {{ _("Total Qty") }}
-    </div>
-    <div class="col-sm-2 text-right">
-        {{ _("Grand Total") }}
-    </div>
+    {% if  doctype == 'Quotation' %}
+        <div class="col-sm-2 ">
+            {{ _("P.O #") }}
+        </div>
+        <div class="col-sm-3 font-size">
+            {{ _("Order Date") }}
+        </div>
+        <div class="col-sm-3">
+            {{ _("Delivery Date") }}
+        </div>
+        <div class="col-sm-2 text-right">
+            {{ _("Qty Ordered") }}
+        </div>
+        <div class="col-sm-2 text-right">
+            {{ _("Total Amount") }}
+        </div>
+    {% elif doctype == 'Sales Order' %}
+        <div class="col-sm-2 ">
+            {{ _("S.O #") }}
+        </div>
+        <div class="col-sm-3 font-size">
+            {{ _("Order Date") }}
+        </div>
+        <div class="col-sm-3">
+            {{ _("Delivery Date") }}
+        </div>
+        <div class="col-sm-2 text-right">
+            {{ _("Total Qty") }}
+        </div>
+        <div class="col-sm-2 text-right">
+            {{ _("Total Amount") }}
+        </div>
+    {% endif %}
```

Comment:
else condition ?

Reply:
You have added the else condition in the header file but nothing in the rows file